src/app/shared/NotePending.svelte

```diff
@@ -62,25 +65,32 @@
 
 <div
   class="loading-bar-content relative flex h-6 w-full items-center justify-between overflow-hidden rounded-md pl-4 text-sm"
-  class:border={!thunk}
+  class:bg-neutral-500={thunk && (isPending || isCompleted)}
+  class:border={!(isPending || isCompleted)}
+  class:px-4={thunk && isPending}
   on:click|stopPropagation>
   {#if thunk && (isPending || isCompleted)}
-    <div class="loading-bar bg-accent" style="width: {$completed}%"></div>
+    <div class="loading-bar bg-accent" style="width: {20 + $completed}%"></div>
     {#if isPending}
       <span>Publishing...</span>
       <span>{total - pendings} of {total} relays</span>
     {:else}
-      <span>Published to {success}/{total} ({failed} failed, {timeout} timed out)</span>
+      <span
+        >Published to {success}/{total}
+        {#if failed > 0 || timeout > 0}
+          ({failed > 0 ? failed + " failed" : ""}{timeout > 0
+            ? (failed > 0 ? ", " : "") + timeout + " timed out"
+            : ""})
+        {/if}
+      </span>
```

Comment:
TERNARY MADNESS

Reply:
🤯
I agree, I'll make a function in the script